app/models/linkset.rb

```diff
@@ -16,6 +16,11 @@ def empty?
   end
 
   def update_attributes_from_gem_specification!(spec)
-    update!(home: spec.homepage)
+    Links::LINKS.except("changelog", "download").each do |short, long|
+      self.attributes = { short => spec.metadata[long] }
+    end
+
+    self.home = spec.homepage if spec.homepage
```

Comment:
sorry.

I tried hard to not end up with a duplicate entry for `homepage` and `home`. But here we are.